src/ansible_navigator/actions/_actions.py

```diff
@@ -111,17 +159,32 @@ def run_interactive(package: str, action: str, *args: Any, **_kwargs: Any) -> An
 
 
 def run_interactive_factory(package: str) -> Callable:
-    """Create a call() function for one package"""
+    """Create a ``run_interactive()`` function for one package.
+
+    :param package: The name of the package
+    :returns: A partial ``run_interactive()`` method for the package
+    """
     return functools.partial(run_interactive, package)
 
 
 def run_stdout(package: str, action: str, *args: Any, **_kwargs: Any) -> Any:
-    """Call the given action's run_stdout"""
+    """Call the given action's ``run_stdout()`` method.
+
+    :param package: The name of the package
+    :param action: The name of the action
+    :param args: The arguments passeed to the action's run_stdout method
+    :param _kwargs: The keyword arguments passed to the action's run_stdout method
+    :returns: The outcome of running the actions' run_stdout method
+    """  # noqa: D402 # Refers to the actions's run_stdout, not this function
```

Comment:
```suggestion
    :returns: The outcome of running the action's ``run_stdout()`` method
    """
```

Reply:
appears the noqa is still needed
```
src/ansible_navigator/actions/_actions.py:171:1: D402 First line should not be the function's "signature"
    """Call the given action's ``run_stdout()`` method.

    :param package: The name of the package
    :param action: The name of the action
    :param args: The arguments passeed to the action's run_stdout method
    :param _kwargs: The keyword arguments passed to the action's run_stdout method
    :returns: The outcome of running the action's run_stdout method
    """
^
1     D402 First line should not be the function's "signature"
1
```

Reply:
Oh, that's why! Then, maybe dropping `()` is fine for the first line.

Reply:
I'd rather have a noqa than one being different...  I'm afraid someone might try to readd it because it was off and no noqa was there to explain

Reply:
Okay, I won't insist. Just don't like having too many noqas. Also, WPS has a rule with a per-module limit of noqas to prevent abuse.